pkgs/development/python-modules/pyobjc-framework-Photos/default.nix

```diff
@@ -11,7 +11,7 @@ buildPythonPackage rec {
     inherit pname version;
     sha256 = "0xgj5h22cijihir97dbwbblkxn55hx5xdicmiwfn8a1lky7mklvd";
   };
-  
```

Comment:
Minor nitpick, but our editorconfig check, we have in github action, usually does not like trailing whitespace. It would be possible to exclude those files so.

Reply:
No longer required. I had some whitespace before the ``<% if...`` in the template file and they where written to the final file.